test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,149 @@
+require "test_helper"
+
+describe UsersController do
+    let (:user) {
+      User.create name: 'chantal'
+    }
+  end
+
+    describe "index" do
+      it "can get the index path" do
+        get users_path
+        must_respond_with :success
+      end
+
+      describe "new" do
+        it "returns status code 200" do
+          get new_user_path
+          must_respond_with :ok
+        end
+      end
+
+      it "creates user" do
+    expect {
+      post users_url, params: { user: {  } }
+    }.must_change "User.count"
+
+    must_redirect_to user_path(User.last)
+  end
+
+  describe "edit" do
```

Comment:
You have the `edit` describe block nested inside the one for `new`. Is that what you intended?